### ISSUE TITLE
SQL assistance applied for single-line comments 

### ISSUE THREAD

**Issue by michal-bartak** [dbeaver/dbeaver#20600]
### Description

In SQL Editor, automatic SQL assistant is run while writing single-line comment. 
In result it makes changes to letters changing them to uppercase. Also, some assistance popups appear.

See video: https://youtu.be/KUqT-pSHDAs

### DBeaver Version

CE 23.1.2

### Operating System

Fedora 38

### Database and driver

1. Database: Postgresql 14
2. Driver: PostgreSQL JDBC Driver v42.5.2

### Steps to reproduce

See video

**Comment by E1izabeth** [comment]
Thank you for bug report.

**Comment by michal-bartak** [comment]
Hello guys. 
I found in the changelog that the fix is provided to v23.1.5.
I've tested it and unfortunately, the DBEaver still behaves incorrectly
See the video
https://youtu.be/IPjIFPnW7G8

Reading name of PR `Fix completion appearing at the end of single line comment line` but it doesn't match the initial issue report. So not sure what has been fixed but for sure not the think I have reported.
